enclave/core/sgx/keys.c

```diff
@@ -362,3 +380,51 @@ oe_result_t oe_get_seal_key_by_policy_v2(
     }
     return result;
 }
+
+oe_result_t oe_get_seal_key_info(
```

Comment:
This method seems like it only exists as an awkward workaround to the existing `oe_get_seal_key/oe_get_seal_key_by_policy` methods to expose SGX-specific sealing key specific concepts at the OE API level that were previously encapsulated. 

Ideally, the user should only have to call `oe_seal`, which takes all relevant inputs, instead of having to do this intermediate step to acquire the `key_info` and if the existing methods to get sealing keys don't expose the appropriate semantics, we should consider deprecating and updating them to some _v3 implementation.